rapid7_insightvm_cloud/plugin.spec.yaml

```diff
@@ -0,0 +1,2237 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: rapid7_insightvm_cloud
+title: Rapid7 InsightVM for the Cloud
+description: InsightVM is a powerful vulnerability management tool which finds, prioritizes, and remediates vulnerabilities. This plugin uses the v4 cloud API to view assets, scan results and start scans
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+status: []
```

Comment:
We know we want this to be cloud enabled in the future so lets set this and make sure we get any validation for cloud_ready run earlier than later.

```suggestion
status: []
cloud_ready: true
```